templates/index.tpl

```diff
@@ -0,0 +1,56 @@
+{**
+ * plugins/generic/backup/templates/index.tpl
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * List of operations this plugin can perform
+ *}
+
+<script src="{$pluginJavaScriptURL}/BackupDownloadHandler.js"></script>
+<script type="text/javascript">
+	$.pkp.plugins.backup.js.BackupDownloadHandler();
```

Comment:
This kind of halfway mimics the PKP javascript framework, which is going to be refactored. I'd favour either following the JS framework patterns completely (so that refactoring doesn't have to take nonstandard code into account), or better yet, avoiding that pattern entirely (e.g. the `$.pkp.plugins.`... namespace) and just embedding the short JS snippet that's required directly in the template here.

Reply:
JS embedded in template.